moveit_ros/planning/rdf_loader/include/moveit/rdf_loader/rdf_loader.h

```diff
@@ -77,6 +85,16 @@ class RDFLoader
     return srdf_;
   }
 
+  void addDescriptionUpdateCallback(StringCallback cb)
```

Comment:
Are these being used anywhere? It would be great to have an example. I assume the callbacks are only meant to be used for initialization and not or updates. Wouldn't separating srdf and urdf updates allow for some very nasty race conditions? If we allow this API, there should at least be a check if URDF and SRDF still fit together.

Reply:
Also, can't the updates be registered with the `StringLoader` directly, so that we can get rid of the extra callback members?

Reply:
A) I added an example using one of the RViz plugins. 
B) I unified it to one callback that only fires when the URDF and SRDF are compatible.